lib/generators/debugs_bunny/migration/migration_generator.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails/generators'
+require 'debugs_bunny/internal/schema/table_descriptor'
+
+module DebugsBunny
+  class MigrationGenerator < Rails::Generators::Base
+    include Rails::Generators::Migration
+
+    class_option :table_name, type: :string, required: true
+
+    source_root File.expand_path('templates', __dir__)
+
+    def generate_migration
+      table_name = options['table_name']
+      table = DebugsBunny::Internal::Schema::TableDescriptor.new(
+        table_name,
+        DebugsBunny::Internal::Schema::ColumnDescriptor.new(:guid, :string, null: false),
+        DebugsBunny::Internal::Schema::ColumnDescriptor.new(:dump, :string, null: false)
```

Comment:
This list of columns will evolve as I add on to this gem, such as support for encrypted dump text.

Reply:
Are the default Rails columns of `created_at` and `updated_at` included here?